examples/demo_mult_image.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
Can you remove this autogenerated header info from this and similar files

Reply:
removed this demo as it was not related to vectors